docs/samples/python/pseudonomyzation.ipynb

```diff
@@ -0,0 +1,356 @@
+    "from presidio_anonymizer import AnonymizerEngine, DeanonymizeEngine, OperatorConfig\n",
+    "from presidio_anonymizer.operators import Operator, OperatorType\n",
+    "\n",
+    "from typing import Dict\n",
+    "from pprint import pprint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 1. Using the `AnalyzerEngine` to identify PII in a text"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "original text:\n",
+      "('Peter gave his book to Heidi which later gave it to Nicole. Peter lives in '\n",
+      " 'London and Nicole lives in Tashkent.')\n",
+      "analyzer results:\n",
+      "[type: PERSON, start: 0, end: 5, score: 0.85,\n",
+      " type: PERSON, start: 23, end: 28, score: 0.85,\n",
+      " type: PERSON, start: 52, end: 58, score: 0.85,\n",
+      " type: PERSON, start: 60, end: 65, score: 0.85,\n",
+      " type: LOCATION, start: 75, end: 81, score: 0.85,\n",
+      " type: PERSON, start: 86, end: 92, score: 0.85,\n",
+      " type: LOCATION, start: 102, end: 110, score: 0.85]\n"
+     ]
+    }
+   ],
+   "source": [
+    "text = \"Peter gave his book to Heidi which later gave it to Nicole. Peter lives in London and Nicole lives in Tashkent.\"\n",
+    "print(\"original text:\")\n",
+    "pprint(text)\n",
+    "analyzer = AnalyzerEngine()\n",
+    "analyzer_results = analyzer.analyze(text=text, language=\"en\")\n",
+    "print(\"analyzer results:\")\n",
+    "pprint(analyzer_results)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2. Creating a custom Anonymizer (called Operator) which replaces each text with a unique identifier.\n",
+    "\n",
+    "To create a custom anonymizer, we need to create a class that inherits from `Operator` and implement the `operate` method. This method receives the original text and a dictionary called `params` with the configuration defined by the user. The method should return the anonymized text.\n",
+    "\n",
+    "In this example we also implement the `validate` method to check that the input parameters are available, i.e. that the `entity_type` and `entity_mapping` parameters are defined, as they are required for this specific anonymizer. `entity_mapping` is a dictionary that maps each entity value to a unique identifier, for each entity type."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class InstanceCounterAnonymizer(Operator):\n",
+    "    \"\"\"\n",
+    "    Anonymizer which replaces the entity value\n",
+    "    with an instance counter per entity.\n",
+    "    \"\"\"\n",
+    "\n",
+    "    REPLACING_FORMAT = \"<{entity_type}_{index}>\"\n",
+    "\n",
+    "    def operate(self, text: str, params: Dict = None) -> str:\n",
```

Comment:
Haha we might want to have this one available as a utility operator.

Reply:
Totally agree, let's add it in a future PR.